src/pages/overview/limits.md

```diff
@@ -2,30 +2,26 @@
 endpoint request (i.e., API call) and the digital output, with each
-operation in the table described below. Content limits apply and are
-provided along with examples for clarity. These limits do not apply to PDF Embed API.
+operation in the table described below. Usage limits apply and are provided along with examples for clarity. These limits do not apply to PDF Embed API.
 
 Paid credentials enable a greater processing quota and require entering
-into a separate written agreement. To learn more, please see our
+into a separate written agreement. To learn more, please see
 [pricing page](/document-services/pricing/#main)
-for purchase options.
-
-Document Transactions for the Operations below resulting in a Document
-will be counted with the following page limits:
+for purchase options.Document Transactions for the Operations below resulting in a Document
+will be counted with the following page limits.
 
 <InlineAlert slots="text"/>
 
-Document Transactions for the Operations below resulting in a Document
-will be counted with the following page limits:<ul><li>Extract Operation: 1 Document Transaction for up to 5 pages</li><li>Accessibility Auto-Tag Operation: 10 Document Transactions per page</li><li>All other Operations: 1 Document Transaction for up to 50 pages</li></ul>
+<ul><li>Extract Operation: 1 Document Transaction for up to 5 pages</li><li>Accessibility Auto-Tag Operation: 10 Document Transactions per page</li><li>All other Operations: 1 Document Transaction for up to 50 pages</li></ul>
 
-### Metrics
+###  Document Transaction Definitions
 
-| Operation           | Capability                                                                                                                                                                                                                     | Metric                               |
+| OPERATION           | CAPABILITY                                                                                                                                                                                                                     | METRIC                               |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------ |
 | Accessibility Auto-Tag | Tag the content in a PDF to improve accessibility                                                                                                                           | 10 Document Transactions (per page)                 |
 | Document Generation | Create PDF and Word documents from your own authored Microsoft Word templates and JSON data                                                                                                                                    | Document Transaction                 |
```

Comment:
Extract PDF -> Extract

Reply:
protect metric definition not updated.

Reply:
Extract PDF calculation table

1 (Operation: Extract PDF) x 2 (5 page document) = 2 Document Transactions
change this to below as per doc
1 (Operation: Extract PDF) x 2 (5-page document) = 2 Document Transactions